src/components/exchange/SwapInfo.js

```diff
@@ -77,28 +81,18 @@ const SwapInfo = ({ asset, amount }) => {
       testID="swap-info"
     >
       <Container>
-        <CoinIcon
-          address={get(asset, 'address')}
-          flex={0}
-          marginRight={5}
-          size={20}
-          symbol={get(asset, 'symbol')}
+        <SwapInfoCoinIcon
+          address={asset?.address}
+          symbol={asset?.symbol}
           testID="swap-info-container"
         />
         <Text color="grey" size="smedium" weight="medium">
           Swapping for{' '}
         </Text>
         <Text color="white" size="smedium" weight="semibold">
-          {`${amountToDisplay}  ${symbol}`}
+          {`${amountToDisplay}  ${asset?.symbol}`}
```

Comment:
is it possible that there's no symbol?

Reply:
yes, `asset` can be `undefined`. technically this component is always being rendered, and its opacity is just being animated